### PR TITLE
Created colors, spacer and font style variables (#1xxujra)

### PR DESCRIPTION
**Pull request by azkyakhan**

### CODE REVIEW

File: css/variables.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+body {
+  /** Typography **/
+  --font-family--heading: "Montserrat", sans-serif;
+  --font-family--subheading: "Open Sans", sans-serif;
```
Comment (dt2patel) [inline css/variables.css]: rename subheading to primary

File: css/variables.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+body {
+  /** Typography **/
+  --font-family--heading: "Montserrat", sans-serif;
```
Comment (dt2patel) [inline css/variables.css]: rename heading to secondary

File: css/variables.css [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  --font-weight--medium: 500;
+  --font-weight--semibold: 600;
+  --font-weight--bold: 700;
+
+  --font-size--xs: 0.75rem;  /* 12px */
+  --font-size--sm: 0.875rem; /* 14px */
+  --font-size--base: 1rem;   /* 16px */
+  --font-size--lg: 1.125rem; /* 18px */
+
+  --h1-font-size: 2.625rem;               /* 42px */
```
Comment (dt2patel) [inline css/variables.css]: customize font sizes to match current website. for example, H1 is currently 56px

File: css/variables.css [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  --spacer-2xl: 5rem;    /* 80px */
+  --spacer-3xl: 10rem;   /* 160px */
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
```
Comment (dt2patel) [inline css/variables.css]: Styles also have to be applied to the individual tags. Currently h6 is not mapped to css variable for h6 font size

**Review by dt2patel — CHANGES_REQUESTED** [review]
Needs more work

**Comment by azkyakhan** [comment]
The branch in this PR was created from main, that is why when I retargeted it automatically get closed. I have done the requested changes in the new PR :
https://github.com/hotwax/omnichannelcto-site/pull/6

File: css/variables.css (hunk 1 above)
Comment (azkyakhan) [reply css/variables.css]: Done

File: css/variables.css (hunk 2 above)
Comment (azkyakhan) [reply css/variables.css]: Done

File: css/variables.css (hunk 3 above)
Comment (azkyakhan) [reply css/variables.css]: Done

File: css/variables.css (hunk 4 above)
Comment (azkyakhan) [reply css/variables.css]: Done

File: css/variables.css (hunk 3 above)
Comment (azkyakhan) [reply css/variables.css]: There are so many font sizes like 11px, 17px 41px etc given to tags. So I have created variables according to approximate values of font sizes used.

**Pull request closed without merging**